system-addon/lib/TopSitesFeed.jsm

```diff
@@ -137,8 +136,12 @@ this.TopSitesFeed = class TopSitesFeed {
       data: this._getPinnedWithData()
     }));
   }
-  onAction(action) {
+  async onAction(action) {
     switch (action.type) {
+      case at.INIT:
```

Comment:
@rlr and I did chat about this and decided that if we do have a `NEW_TAB_LOAD` it seemed undesirable for it to be blocked on a potentially unbounded amount of time for `tippyTopProvider.init`